db/migrations/public/20240430133702_water-add-licence-agreements-view.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const viewName = 'licence_agreements'
+
+exports.up = function (knex) {
+  return knex
+    .schema
+    .createView(viewName, (view) => {
+      view.as(knex('licence_agreements').withSchema('water').select([
+        'licence_agreement_id AS id',
+        'financial_agreement_type_id',
```

Comment:
As our view name is different from the table name we'll need to alias this field as well.

```suggestion
        'financial_agreement_type_id AS financial_agreement_id',
```

Reply:
Is it weird that the change didn't break any of the tests?

Reply:
ah it failed on the pipeline so I've done something wrong locally when tryign to redo the migrations